frontend/src/App.js

```diff
@@ -176,15 +202,15 @@ class App extends Component {
             grams.push(item.grams);
         }
 
-        return {totalImpact: json["impact"] / 1000 * 2.2, // Convert from grams to pounds
+        return {totalImpact: totalImpact,
                 contributors: contributors,
                 impacts: impacts,
                 grams: grams,
                 driveEq: json["drive_eq"],
                 totalLandUse: totalLandUse,
                 // parkingEq: totalLandUse / 14, // Convert from sq meters to # parking spots
                 totalTreeUse: totalLandUse / 256, // 256sqft/tree Convert from land use to Californian tree use
-                totalWaterUse: totalWaterUse * 4.2, // Water use in cups
+                totalWaterUse: totalWaterUse, // Water use in cups
                 // totalWaterUse: totalWaterUse, // Water in liters
                 totalShowerUse: totalWaterUse / 65.1 // Converts water use to shower use (65.1 liters per shower)
```

Comment:
```suggestion
                totalShowerUse: totalWaterUse / (65.1 * 4.2) // Converts water use to shower use (65.1 liters per shower)
```
I see totalWaterUse was converted from liters to cups earlier,  but since totalShowerUse is calculated based on 65.1 liters/shower, we need to update this to the proper calculation in cups/shower.

Also, delete the now irrelevant and incorrect comment `// totalWaterUse: totalWaterUse, // Water in liters`